pkg/config/base_path_test.go

```
@@ -0,0 +1,134 @@
+		BasePathFromArg: ".",
+	}
+
+	expectedPath, _ := filepath.Abs(".")
+
+	result, err := configLoader.BasePathComputing(info)
+
+	assert.NoError(t, err)
+	assert.Equal(t, expectedPath, result)
+	// test non-existent base path
+	info.BasePathFromArg = "invalid/path"
+	expectedPath, _ = filepath.Abs("invalid/path")
```

golangci-lint (1.62.2)

ineffectual assignment to expectedPath

(ineffassign)